Dockerfile

```
@@ -64,14 +65,22 @@ ENV PORT= \
     AUTO_UPDATE_WARN_MINUTES=30 \
     AUTO_REBOOT_ENABLED=false \
     AUTO_REBOOT_WARN_MINUTES=5 \
-    AUTO_REBOOT_CRON_EXPRESSION="0 0 * * *"
+    AUTO_REBOOT_CRON_EXPRESSION="0 0 * * *" \
+    DISCORD_CONNECT_TIMEOUT=30 \
+    DISCORD_MAX_TIMEOUT=30 \
+    DISCORD_PRE_UPDATE_BOOT_MESSAGE="Server is updating..." \
+    DISCORD_POST_UPDATE_BOOT_MESSAGE="Server update complete!" \
+    DISCORD_PRE_START_MESSAGE="Server is started!" \
+    DISCORD_PRE_SHUTDOWN_MESSAGE="Server is shutting down..." \
+    DISCORD_POST_SHUTDOWN_MESSAGE="Server is stopped!"
 
 COPY ./scripts/* /home/steam/server/
 
 RUN chmod +x /home/steam/server/*.sh && \
     mv /home/steam/server/backup.sh /usr/local/bin/backup && \
     mv /home/steam/server/update.sh /usr/local/bin/update && \
-    mv /home/steam/server/restore.sh /usr/local/bin/restore
+    mv /home/steam/server/restore.sh /usr/local/bin/restore && \
+    mv /home/steam/server/discord.sh /usr/local/bin/discord
```

```suggestion
    mv /home/steam/server/discord.sh /usr/local/bin/discord
```

I don't think we should make discord a command. I would recommend calling the script directly.

---

good idea, makes sense